k8s/cloud/base/proxy_deployment.yaml

```diff
@@ -78,6 +78,36 @@ spec:
         volumeMounts:
         - name: certs
           mountPath: /certs
+        - name: nginx-config
```

Comment:
I think you should be able to mount the configmap as a directory of files without having to specify each file.

Reply:
Good call. Verified that mounting the entire directory worked 👍